unocounter/app/components/Calculator.tsx

```
@@ -0,0 +1,197 @@
+  return (
+    <div className="fixed inset-0 bg-black bg-opacity-50 flex items-center justify-center z-50">
+      <div className="bg-white rounded-lg shadow-xl p-6 w-72">
+        <div className="mb-4 text-right bg-gray-100 p-3 rounded text-2xl font-mono overflow-x-auto">
+          {display}
+        </div>
+
+        <div className="grid grid-cols-4 gap-2 mb-4">
+          <button
+            onClick={handleClear}
+            className="col-span-1 bg-red-100 text-red-800 p-3 rounded font-bold hover:bg-red-200"
+          >
```

Biome (2.1.2)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)